tests/publiccloud/slem_basic.pm

```diff
@@ -25,13 +25,42 @@ sub has_wicked {
     return 0;
 }
 
+sub check_avc {
```

Comment:
maybe better to create dedicated SELinux test ( and also take all other SELinux dedicated logic )  ?  It starts to remind me console_tests where people were throwing a lot of different things until it become so big that other people needed spent time on refactoring all this into separate test suites 
 This one is still called `slem_basic` but keep moving out from this definition ...

Reply:
This test contains the most basic SELinux tests since the beginning of the test runs. This is just an extension to them.

Reply:
I don't want to start long thread here , you may keep it like this if you want . I will just give you last reason if you think it is not worth it anyway let's keep it like this : 
- reporting . currently in case of any failure you just see that slem_basic read . when you will create slem_selinux you will see if it is exactly SELinux related problem or common .

But I am resolving conversation because don't want to push you here

Reply:
My answer is not a "no", my answer is a "not yet", or in other words - not the scope of this PR :slightly_smiling_face: